README.md

```diff
@@ -128,45 +136,46 @@ message = %{"id" => "tx-1", "amount" => 15.99}
   124, 66, 49, 157, 51, 242, 3, 33, 52, 161, 147, 221, 174, 114, 48>>
 ```
 
-If you want to controll output format, you can provide `:format` option.
-Possible values are:
+The `:format` argument controls output format:
 
-- `:ocf` - embeds schema with [Object Container Files](https://avro.apache.org/docs/1.8.1/spec.html#Object+Container+Files) format
-- `:registry` - embeds Confluent [Schema Registry](https://docs.confluent.io/current/schema-registry/serializer-formatter.html#wire-format) magic version
-- `:plain` - only encode message with nothing embeded
-- `:guess` - fallbacks to `:ocf` if can't behave like `:registry` _(default)_
+  * `:plain` - Just return Avro binary data, with no header or embedded schema
+  * `:ocf` - Use [Object Container File]https://avro.apache.org/docs/1.8.1/spec.html#Object+Container+Files)
+    format, embedding the full schema with the data
+  * `:registry` - Write data with Confluent Schema Registry
+    [Wire Format](https://docs.confluent.io/current/schema-registry/serializer-formatter.html#wire-format),
+    which prefixes the data with the schema id
+  * `:guess` - Use `:registry` if possible, otherwise use `:ocf` (default)
 
 ```elixir
 message = %{"id" => "tx-1", "amount" => 15.99}
 
-{:ok, pid} = Avrora.start_link()
```

Comment:
I think if I just install the library by adding it to the `mix.exs` I need to start memory storage (if no supervision tree was used). And if I copy/paste this block of the code it will not work

Reply:
Yes, probably better to keep `Avrora.start_link` with each example so that they can be typed in. 

Generally speaking, I think it's better to have users add the server to their own supervision tree. In this case, it's less of an issue.

Reply:
> Generally speaking, I think it's better to have users add the server to their own supervision tree. In this case, it's less of an issue.

No doubts, you are right.

But code samples suppose to be the samples or examples which you can copy-paste and be happy. Also `doctests` are doing exactly the same, they are testing that an example you have will work.

I think that's the only concern I have and if you can change it back, we can merge it 🤝

Reply:
I have changed it back.